pkgs/development/libraries/plplot/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchzip, cmake }:
+
+let pname = "plplot"; in
```

Comment:
Don't use `pname` like that. You don't even need the `let <>; in` syntax here. Just put the `pname` inside the `mkDerivation`.

Reply:
Resolved. (I had done that to make it non-recursive but I forgot to do so ...)